lib/web_ui/lib/src/engine/canvaskit/image.dart

```diff
@@ -47,11 +47,24 @@ class CkAnimatedImage implements ui.Image {
     box = SkiaObjectBox(this, _skAnimatedImage as SkDeletable);
   }
 
+  bool _disposed = false;
   @override
   void dispose() {
     box.delete();
+    _disposed = true;
   }
 
+  @override
+  bool get debugDisposed {
+    bool? disposed;
+    assert(() {
```

Comment:
On the web side we prefer using `assertionsEnabled`.

Reply:
Done